### ISSUE TITLE
Page alert, role="alert" needs to be removed and documented

### ISSUE THREAD

**Issue by alex-page** [govau/uikit#569]
# Bug Report

- [x] I’ve read and understood the [Contributing guidelines](https://github.com/govau/uikit/blob/master/CONTRIBUTING.md) and have done my best effort to follow them.
- [x] I’ve read and agree to the [Code of Conduct](https://github.com/govau/uikit/blob/master/CODE_OF_CONDUCT.md).
- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

The `role="alert"` is a way to alert a user that something has changed. If the user is using a screen reader it is the first thing the user will hear on the page. It is an assertive way to update the user of important information.

## What I expected to happen

The role should be removed and we should add some guidance on when and how to correctly use the role. In the future when we have inline form errors we may discuss adding the role back in.

## Reproducing

- Module name: page-alerts
- Module version: 2.0.8
